packages/components/src/textarea-control/index.tsx

```diff
@@ -57,7 +58,7 @@ export function TextareaControl(
 	const id = `inspector-textarea-control-${ instanceId }`;
 	const onChangeValue = ( event: ChangeEvent< HTMLTextAreaElement > ) =>
 		onChange( event.target.value );
-
+	const textareaRef = useRef( null );
```

Comment:
We need to get the forwarded ref from the function arguments, not create a new ref. You can check the [`InputControl` implementation](https://github.com/WordPress/gutenberg/blob/77ee6e988181e15ebb3cb099891822f16df9808d/packages/components/src/input-control/index.tsx#L55) to see how that works.